### ISSUE TITLE
Intermediate logits's contribution？

### ISSUE THREAD

**Issue by CuberrChen** [DeepSceneSeg/EfficientPS#12]
hi! The design of the network is very clever and it has achieved good results! But what I don't understand is why intermediate logits in fusion modeul are needed. According to my understanding，`argmax(intermediate logits)` actually represents that the possible instance part（In addition to the fusion results） of the semantic logits are accepted. 
Hope to get your reply！
good luck.

**Comment by mohan1914** [comment]
The additional step is required to work around the sensitivity of the PQ metric by removing stuff classes that have very few pixels otherwise those pixels will be treated as false positives. The additional step improves the PQ metric a bit with respect to stuff classes.
